scripts/release-notes/tests/test_aggregate_release_notes.py

```
@@ -0,0 +1,873 @@
+            # Skipped
+            _make_issue_with_adf(
+                "DAT-105", "canonical_h2", issuetype="Story", summary="Internal",
+                labels=["skipReleaseNotes"],
+            ),
+            # No H2, no skip
+            _make_issue_with_adf("DAT-106", "no_h2", issuetype="Bug", summary="No note bug"),
+        ]
+        return accumulate_issues(issues)
+
+    def test_render_notes_grouped_by_issuetype(self) -> None:
+        entries, buckets, _proj_counts = self._build_mixed_entries_and_buckets()
```

Ruff (0.15.14)

[warning] Unpacked variable `buckets` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)